packages/web-components/src/components/rux-dialog/rux-dialog.tsx

```diff
@@ -186,9 +187,14 @@ export class RuxDialog {
     }
 
     private _handleSlotChange() {
+        console.log('handle slot change trigger')
```

Comment:
think these are failing the tests

Reply:
yeah, I pushed this up just for Kiley to see. Will clean it up